src/components/recordingcreator/recordingfields.js

```diff
@@ -215,7 +204,7 @@ define(['globalize', 'connectionManager', 'serverNotifications', 'require', 'loa
                 });
             }
         } else {
-            showSingleRecordingFields(options.parent, options.programId, apiClient);
+            context.querySelector('.recordSeriesContainer').classList.add('hide');
```

Comment:
@cvium I am on the fence about this line since it wasn't explicitly added in `showSingleRecordingFields`

Reply:
I don't think it's necessary. We can just have some of the live tv folks test the RC and re-add it if it looks funny to them.